works.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="works-style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+    <title>works</title>
+</head>
+
+<body>
+    <section class="wrapper">
+        <div class="card">
+            <div class="snapshot"><img src="images/card1.svg" alt="tonic"></div>
+            <div class="block">
+                <div class="title">Tonic</div>
+                <div class="subfram">
+                    <div class="sub1">CANOPY</div>
+                    <div class="circle"></div>
+                    <div class="sub2">Back End Dev</div>
+                    <div class="circle"></div>
+                    <div class="sub2">2015</div>
+                </div>
+                <div class="normal">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                <div class="tags">
+                    <div class="tag">
+                        <div class="tagtxt">html</div>
+                    </div>
+                    <div class="tag">
+                        <div class="tagtxt">css</div>
+                    </div>
+                    <div class="tag">
+                        <div class="tagtxt">javaScript</div>
+                    </div>
+                </div>
```

Comment:
- [ ] This list of project technologies should be a list `ul` and each of them a `li` pointed out from the requirement of this project. Can we fix it?.